UPGRADING.md

```diff
@@ -26,6 +26,21 @@ This guide provides instructions for upgrading to specific versions of CometBFT.
   `mempool_error`, which were only used by the priority mempool, were removed
   but still kept in the message as "reserved".
 
+
+### KvIndexer changes
+
+When indexing block events, the indexer would include the function that returned those events
+(`BeginBlock` or `EndBlock`). In ABCI 2.0 `FinalizeBlock` combines these functions, and all
+events are returned within `ResultFinalizeBlock`. Therefore, the function field will
+have no meaning anymore, and to reduce storage space taken by the indexer,
+it will be removed in CometBFT v0.38.x.
+
+Events indexed with previous CometBFT or Tendermint versions, will still be processed, 
+but the function field is ignored. There is no need to re-index the events.
+
+However, queries checking for this field, will return nothing. 
```

Comment:
Hmm not sure I understand what it means. I think it means that the event indexer schema remains unchanged, but one of the columns/field in the schema (called `function`) will be empty starting with v0.38.

The way I understand the purpose of this section is: _as a user (eg, Marko, Zohar, Jay, Mircea, Luca), do I need to do anything when I upgrade to v0.38?_

By the way, I'm not technically advanced, so I might be a good example of a novice/intermediate user of CometBFT.

> However, queries checking for this field, will return nothing. 

Which RPC queries exactly, is it `/block_search`?

Reply:
Got it, makes sense!